moveit_ros/planning/moveit_cpp/src/planning_component.cpp

```diff
@@ -177,6 +185,8 @@ PlanningComponent::PlanSolution PlanningComponent::plan(const PlanRequestParamet
   }
   last_plan_solution_->start_state = req.start_state;
   last_plan_solution_->trajectory = res.trajectory_;
```

Comment:
Since it's pretty much the exact same data anyway, could you replace the `PlanSolution` struct with MotionPlanResponse` from [here](https://github.com/ros-planning/moveit2/blob/main/moveit_core/planning_interface/include/moveit/planning_interface/planning_response.h#L46), and maybe add the bool conversion to it? The start_state is not needed, really.

Reply:
Good idea